qiskit/algorithms/__init__.py

```diff
@@ -384,6 +397,7 @@
     "HamiltonianPhaseEstimation",
     "HamiltonianPhaseEstimationResult",
     "VQD",
+    "VQDResult",
```

Comment:
Although this was not in `all` before it was also not in the docs above - maybe if we are adding it here we should add it there for now too.